howso/engine/client.py

```diff
@@ -1,5 +1,5 @@
-from howso.client import AbstractHowsoClient, HowsoPandasClient
-from howso.client.pandas import HowsoPandasClientMixin
+import howso.client
```

Comment:
Should be able to revert these import changes. There should be no circular import with the clients.

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60

Reply:
Still missing type hints on `use_client`